server/service/src/pack_variant/query.rs

```diff
@@ -19,7 +19,8 @@ pub fn get_pack_variants(ctx: &ServiceContext) -> Result<Vec<ItemPackVariant>, R
     let stock_on_hand = StockOnHandRepository::new(connection).query(Some(
         StockOnHandFilter::new().store_id(EqualFilter::equal_to(store_id)),
     ))?;
-    let pack_variants = PackVariantRowRepository::new(connection).load_all()?;
+    let pack_variants = PackVariantRepository::new(connection)
+        .query_by_filter(PackVariantFilter::new().is_active(true))?;
 
     // Calculate the most used variant for each item and pack size by total number of packs
     // if item has stock on hand else by empty line count. HashMap keys are (item_id, pack_size)
```

Comment:
Sorry I neve ended up looking at the code below (to get most used).

We will probably need to change it a little bit, to use SQL and let the database do the work (for performance).

Something like:
```sql
CREATE VIEW pack_variant_stats as
SELECT sum(count(invoice_line) + count(stock_line)) as number_of_lines, pack_variant.id as pack_variant_id 
FROM pack_variant
LEFT JOIN stock_line on pack_variant.pack_size = stock_line.pack_size AND pack_variant.item_id = stock_line.item_id AND stock_line.available_number_of_packs > 0
LEFT JOIN invoice_line on pack_variant.pack_size = invoice_line.pack_size AND pack_variant.item_id = invoice_line.item_id
LEFT JOIN invoice on invoice_line.invoice_id = invoice.id 

WHERE stock_line.store_id = {store_id} and invoice.store_id = {store_id}

GROUP BY pack_variant.id

ORDER BY pack_variant_id, number_of_lines
```

Above should be pretty easy to map in rust